base_layer/wallet/src/base_node_service/service.rs

```diff
@@ -48,6 +48,7 @@ pub struct BaseNodeState {
     pub is_synced: Option<bool>,
     pub updated: Option<NaiveDateTime>,
     pub latency: Option<Duration>,
+    pub status: OnlineStatus,
```

Comment:
Oh I see I misinterpreted this in the previous iteration of this PR as being a struct that lives in the Console Wallet, It is definitely not right to duplicate this state in a separate service.